product/host-input/fw/config_power_domain.c

```diff
@@ -0,0 +1,175 @@
+/*
+ * Arm SCP/MCP Software
+ * Copyright (c) 2017-2019, Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdint.h>
+#include <string.h>
+
+#include <fwk_element.h>
+#include <fwk_log.h>
+#include <fwk_macros.h>
+#include <fwk_mm.h>
+#include <fwk_module.h>
+#include <fwk_module_idx.h>
+#include <power_domain_utils.h>
+
+//#include <mod_system_power.h>
+#include <mod_power_domain.h>
+//#include <mod_ppu_v1.h>
+
+#include "config_power_domain.h"
+#include "config_ppu_v0.h"
+
+/* Mask of the allowed states for the systop power domain */
```

Comment:
remove commented code or else provide justification in a comment here and everywhere below